languages/php/src/BitwardenClient.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Bitwarden\Sdk;
+
+use Bitwarden\Sdk\Schemas\AccessTokenLoginRequest;
+use Bitwarden\Sdk\schemas\ClientSettings;
+use Bitwarden\Sdk\Schemas\Command;
+use FFI;
+use Swaggest\JsonDiff\Exception;
+
+
+class BitwardenClient
+{
+    private BitwardenLib $bitwarden_lib;
+
+    private ClientSettings $clientSettings;
+
+    public ProjectsClient $projects;
+
+    public SecretsClient $secrets;
+
+    private CommandRunner $commandRunner;
+
+    private FFI\CData $handle;
+
+    public function __construct(ClientSettings $clientSettings)
+    {
+        $this->clientSettings = $clientSettings;
+        $this->clientSettings->apiUrl = getenv('API_URL') ?: 'https://api.bitwarden.com';
+        $this->clientSettings->identityUrl = getenv('IDENTITY_URL') ?: 'https://identity.bitwarden.com';
+        $this->clientSettings->userAgent = getenv('USER_AGENT') ?: 'SDK';
+        $this->clientSettings->deviceType = getenv('DEVICE_TYPE') ?: 'SDK';
```

Comment:
Now that we're passing the clientSettings as a parameter we don't need to set these here, so these lines can be removed.

The user should be the one in charge of setting these values, like the code in the readme shows.

Reply:
It's removed now. Sorry, I was swayed by implementations in other languages.